pkg/kubectl/resource_printer.go

```diff
@@ -446,9 +445,9 @@ func (h *HumanReadablePrinter) HandledResources() []string {
 	return keys
 }
 
-func (h *HumanReadablePrinter) FinishPrint(output io.Writer, res string) error {
-	if !h.options.NoHeaders && !h.options.ShowAll && h.hiddenObjNum > 0 {
-		_, err := fmt.Fprintf(output, "  info: %d completed object(s) was(were) not shown in %s list. Pass --show-all to see all objects.\n\n", h.hiddenObjNum, res)
+func (h *HumanReadablePrinter) FinishPrint(output io.Writer, res string, hiddenObjNum int) error {
```

Comment:
Seems like a really high cost (and a lot of literal `0` getting passed into `FinishPrint()` impls for this -- maybe there's a way to do this with less effort? The Printers no longer should care about filtering and this keeps a tenuous connection there